js/utils/modalManager.js

```diff
@@ -57,6 +59,23 @@ export function launchSettingsModal(modalOptions = {}) {
   return aboutModal;
 }
 
+export function launchPurchaseModal(modalOptions = {}) {
```

Comment:
Can you just launch this modal from the listing details view? Modal manager is more designed for singleton modals where an instance is shared app wide.

This guy could really just be launched from wherever it's CTA is. Please be sure to remove the modal in the remove of the view that launches it.

moderatorDetailsModal would fall into the same category, but I think we could leave it be for now. It wasn't a 100% clear to me the exact role of modalManager, so I didn't flag it earlier.